clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/isolated/IsolatedIngestionServer.java

```diff
@@ -769,9 +774,17 @@ private void initializeIsolatedIngestionServer() {
     storeIngestionService.addIngestionNotifier(new IsolatedIngestionNotifier(this));
     ingestionBackend = new DefaultIngestionBackend(storageMetadataService, storeIngestionService, storageService);
 
-    LOGGER.info(
-        "Starting report client with target application port: {}",
-        configLoader.getVeniceServerConfig().getIngestionApplicationPort());
+    if (serverConfig.isLeakedResourceCleanupEnabled()) {
```

Comment:
Also, it suddenly comes to me that -> Is Helix guaranteed to issue DROP message for old resources if a node is down for enough time? For now we can assume that, but if this is not the case then the fix in main process to drop resource if NOT_EXIST is not enough I guess..(then we will need to restore here, and try to rely on LeakedResourceCleaner to clean old storage engine). It is my concern, but I might be wrong though.

Reply:
Just my thinking. If Helix is guaranteed to send DROP as I mentioned above, then data partition is guaranteed to be dropped and this PR will guarantee metadata partition will also be deleted, that's good enough. 

The only lingering would be existing leaks. My take is it is OK for now.

Reply:
Thanks for the review. I have verified with Helix team, if a node goes down for a period longer than rebalance delay period, Helix will select a new replica and when the node comes up online later it will see drop messages. So our main process deletion PR should work.

Reply:
Okay. I see. if this is true, then it should be good. If we found out that it does not work as Helix said, we should let them know and try to go back to original proposal (restore in II and rely on cleaner to clean up).